lua/gitlab/actions/discussions/init.lua

```diff
@@ -398,14 +398,14 @@ local function get_new_line(node)
 
   ---@type GitlabLineRange|nil
   local range = node.range
-  if range == nil then
-    if node.new_line == nil then
+  if range ~= nil then
+    if range.start.new_line == nil then
       return nil
     end
-    return node.new_line
+    return range.start.new_line
   end
 
-  local start_new_line, _ = common.parse_line_code(range.start.line_code)
+  local _, start_new_line = common.parse_line_code(range.start.line_code)
   return start_new_line
```

Comment:
I've applied what you suggested. But I kept using the `range.start.new_line` (and `old_line`), because I don't see why not use them if available. Unless you plan to remove them in the future? Or is there a problem/subtility I ddn't saw with them?

Reply:
Yeah believe it or not I've actually found that the two can be different -- I'm not sure why, but Gitlab's APIs seem to be inconsistent on the API response in terms of what they put in that range information and what the hash of the line is itself. I've run into issues with using the range info on the node directly in the past